.github/workflows/ci-s390x.yaml

```diff
@@ -0,0 +1,36 @@
+name: Operator E2E Test for s390x
+on:
+  workflow_call:
+
+permissions:
+  contents: read
+
+jobs:
+  operator-e2e:
+    runs-on: s390x
+
+    strategy:
+      matrix:
+        runtimeclass: ["kata-qemu"]
+
+    steps:
+      - name: Adjust a permission for repo
+        run: |
+          sudo chown -R $USER:$USER $GITHUB_WORKSPACE
```

Comment:
hi @BbolroC should this chown call be part of the pre-action too?

Reply:
Yeah, you are totally right. Thanks for pointing it out. I will update the PR.

FYI: this optimisation will also apply to kata GHA (https://github.com/kata-containers/kata-containers/pull/8649) soon. 😉